pynestml/utils/model_installer.py

```diff
@@ -61,23 +73,30 @@ def install_nest(target_path: str, nest_path: str) -> None:
     else:
         shell = False
 
+    # remove CMakeCache.txt if exists
+    cmake_cache = os.path.join(target_path, "CMakeCache.txt")
+    if os.path.exists(cmake_cache):
+        os.remove(cmake_cache)
+
     # first call cmake with all the arguments
     try:
-        result = subprocess.check_call(cmake_cmd, stderr=subprocess.STDOUT, shell=shell,
-                                       cwd=str(os.path.join(target_path)))
+        result = subprocess.check_call(cmake_cmd, stderr=subprocess.STDOUT,
+                                       shell=shell, cwd=str(os.path.join(target_path)))
```

Comment:
What does the `join` on the `target_path` do here anyway? And shouldn't that be a string already?